tooling/cli/schema.json

```diff
@@ -1500,7 +1500,7 @@
           ]
         },
         "sidecar": {
-          "description": "Enable sidecar execution, allowing the JavaScript layer to spawn a sidecar command, an executable that is shipped with the application. For more information see <https://tauri.studio/docs/guides/bundler/sidecar/>.",
+          "description": "Enable sidecar execution, allowing the JavaScript layer to spawn a sidecar command, an executable that is shipped with the application. For more information see <https://tauri.studio/guides/building/sidecar>.",
```

Comment:
Add `v1` to url

Reply:
Yeah we need to go through the whole codebase for that one before v1 stable. Although there shouldn't be many more occurrences i think.